text/0012-CLI-commands.md

```diff
@@ -0,0 +1,1367 @@
+the main public collection of datasets & peers. "qri new" and "qri update"
+default to publishing to a registry as part of dataset creation unless run 
+with the "no-publish" flag. Published datasets that are under 20MBs will be
+backed up by the Qri registry.
+
+Unpublished dataset info will be held locally so you can still interact
+with it. And your datasets will be available to others peers when you run 
+"qri connect", but will not show up in search results, and will not be 
+displayed on lists of registry datasets.
+
+Qri is designed to work without a registry should you want to opt out of
+centralized listing entirely, but know that peers who *do* participate in
+registries may choose to deprioritize connections with you. Opting out of a
+registry entirely is better left to advanced users.
+
+You can opt out of registries entirely by running:
+$ qri config set registry.location ""
+
+### Options
+
+```
+  -h, --help   help for registry
+```
+
+
+
+<a id='qri_registry_publish'></a>
+## qri registry publish
+
+Publish dataset info to the registry
+
+### Synopsis
+
+Publishes the dataset information onto the registry. There will be a record
+of your dataset on the registry, and if your dataset is less than 20mbs, 
+Qri will back your dataset up onto the registry.
+
+Published datasets can be found by other peers using the `qri search` command.
+
+Datasets are by default published to the registry when they are created.
+
+```
+qri registry publish [flags] <dataset_reference>
+```
+
+### Examples
+
+```
+  Publish a dataset you've created to the registry:
+  $ qri registry publish me/dataset_name
+```
+
+### Options
+
+```
+  -h, --help   help for publish
+```
+
+
+
+<a id='qri_registry_unpublish'></a>
+## qri registry unpublish
+
+Remove dataset info from the registry
+
+### Synopsis
+
+Unpublish will remove the reference to your dataset from the registry. If 
+you dataset was previously backed up onto the registry, this backup will 
+be removed.
+
+This dataset will no longer show up in search results. 
+
+```
+qri registry unpublish [flags] <dataset_reference>
+```
+
+### Examples
+
+```
+  Remove a dataset from the registry:
+  $ qri registry unpublish me/dataset_name
+```
+
+### Options
+
+```
+  -h, --help   help for unpublish
+```
+
+
+
+<a id='qri_remove'></a>
+## qri remove
+
+remove a dataset from your local repository
+
+### Synopsis
+
+
+Remove gets rid of a dataset from your Qri node. After running remove, Qri will 
+no longer list your dataset as being available locally. By default, remove frees
+up the space taken up by the dataset, but not right away. The IPFS repo that’s 
+storing the data will need to garbage-collect that data when it’s good & ready, 
+which could be anytime. If you’re running low on space, garbage collection will 
+be sooner. 
+
+Keep in mind that by default your IPFS repo is capped at 10GB in size, if you
+adjust this cap using IPFS, Qri will respect it.
+
```

Comment:
Am I wrong in sussing that this is an IPFS type command. Can we give guidance on how to change it? Or a way to change this using the Qri config?

Reply:
We should def create documentation on how to change it as part of a larger "configuring IPFS" documentation, and reference it again in another set of docs on "managing Qri Data". I'll open issues over on the website repo to create these.